### PR TITLE
Fixed missing tag of pewter bowl and its description

### PR DESCRIPTION
**Pull request by cainiaowu**

### CODE REVIEW

**Comment by illi-kun** [comment]
Why it should be sealed & watertight? I don't think it has a container's cap.

**Comment by KrazyTheFox** [comment]
> I don't think it has a container's cap.

I'm pretty sure I've never seen nor heard of a pewter bowl that's watertight. Google images seems to back that up: https://www.google.com/search?q=pewter+bowl&sa=X&biw=1920&bih=982&tbm=isch&tbo=u&source=univ&ei=YyVEVYqtOcTAggS9-YK4Cg&ved=0CB4QsAQ

**Comment by cainiaowu** [comment]
You need these 2 tags to hold liquid as the description claimed.

File: data/json/items/containers.json [hunk 1]
```diff
@@ -112,7 +112,10 @@
         "to_hit":-1,
         "material":"steel",
         "contains":1,
-        "rigid": true
+        "rigid": true,
+        "watertight": true,
+        "seals": true,
+        "qualities": [["CONTAIN", 1]]
```
Comment (Rivet-the-Zombie) [inline data/json/items/containers.json]: Adding this quality is good. All the rest needs to go; it's a bowl, not a bottle.

Comment (cainiaowu) [inline data/json/items/containers.json]: well, please compare this with plastic bowl(line 568) and clay bowl(line 761). They both have these flags.

Comment (kevingranade) [inline data/json/items/containers.json]: Plastic bowl I think is supposed to be tupperware with a lid, I'm not sure about the clay bowl but it probably also shouldn't have a lid.

Comment (kevingranade) [inline data/json/items/containers.json]: If you look at the recipe for making the clay bowls, they have waterproofing materials and a fur or leather lid, the petwer bowl doesn't have a lid.

**Pull request merged**